Templates/QueueTrigger-Powershell/metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "defaultFunctionName": "QueueTriggerPowershell",
+    "defaultFunctionName": "QueueTriggerPowerShell",
     "description": "A PowerShell function that will be run whenever a message is added to a specified Azure Storage queue",
     "name": "QueueTrigger - PowerShell",
-    "language": "Powershell",
+    "language": "PowerShell",
```

Comment:
I did it in all templates. I did a find in files for "Powershell". It is no more.